src/lib.rs

```diff
@@ -333,6 +334,12 @@ pub struct RenderPipelineDescriptor<'a> {
 
     /// The number of samples calculated per pixel (for MSAA).
     pub sample_count: u32,
+
+    /// Bitmask deciding how much of the samples are kept.
+    pub sample_mask: u32,
+
+    /// When enabled, the sample_mask is ANDed with the the alpha value.
```

Comment:
When enabled, produces another sample mask per pixel based on the alpha output value, that is ANDed with the `sample_mask` and the primitive coverage to restrict the set of samples affected by a primitive.

The implicit mask produced for alpha of zero is guaranteed to be zero, and for alpha of one is guaranteed to  be all 1-s.

Reply:
Thanks!